src/Language/ar/Auth.php

```diff
@@ -30,7 +30,7 @@
     'password'        => 'كلمة المرور',
     'passwordConfirm' => 'كلمة المرور (مرة اخرى)',
     'haveAccount'     => 'هل لديك حساب بالفعل؟',
-    'token'           => '(To be translated) Token',
+    'token'           => 'رمز Token',
```

Comment:
```suggestion
    'token'           => 'رمز',
```


_"Token"_ is English, and should not exist here

Reply:
@mshannaq Can you update?